### ISSUE TITLE
may you share link  on mentioned your habr post about validation? 

### ISSUE THREAD

**Issue by Sandy4321** [Danila89/cross_validation_custom#1]
may you share link  on mentioned your habr post about validation?  
https://youtu.be/Otw9MAkXQD4?t=15
Мифы про интерпретацию моделей – Данила Савенков

may it has new tile 
https://habr.com/en/company/ods/blog/336168/
Kaggle Mercedes и кросс-валидация

**Comment by Danila89** [comment]
Yes, this https://habr.com/en/company/ods/blog/336168/ is exactly the link I mentioned in the video

**Comment by Sandy4321** [comment]
great thanks, by the way , what to do if we need more folds? shuffling may give very similar data chunks?

**Comment by Sandy4321** [comment]
did you tried one out? 
https://statmodeling.stat.columbia.edu/2018/06/05/comments-limitations-bayesian-leave-one-cross-validation-model-selection/
Comments on Limitations of Bayesian Leave-One-Out Cross-Validation for Model Selection

**Comment by Danila89** [comment]
I can't imagine the situation where we will need really a lot of folds. Very small folds tend to give higher variance and there is no sense to increase the number of folds to much. In general more folds you have - less times you should do cross-validation. The limit case if leave-one-out where one should do this only once.
I have not tried Bayesian leave-one-out
